ocaml/parsing/ast_iterator.mli

```diff
@@ -71,6 +71,8 @@ type iterator = {
   structure_item_jane_syntax: iterator -> Jane_syntax.Structure_item.t -> unit;
   typ: iterator -> core_type -> unit;
   typ_jane_syntax: iterator -> Jane_syntax.Core_type.t -> unit;
+  constructor_argument_jane_syntax: iterator ->
```

Comment:
I considered it, but decided against it because `constructor_argument` is only a Jane-syntax notion, the regular AST doesn't distinguish it from types.  I'm not sure what would even really go in a `constructor_argument` function.  Thoughts?

Reply:
That seems reasonable enough, and I don't feel too strongly about this (just wanted to make sure it was considered), so I'll resolve this.